### DISCUSSION TITLE
Anones Actions also stuck ?

### DISCUSSION THREAD

**Discussion by jimmybsilva** [community/community#121047] [Actions]
### Select Topic Area

Question

### Body

 22 minutes ago Queued and still has not ran usually takes seconds?

**Comment by cmanfre4** [comment]
Yes, my jobs are not currently being picked up.

Edit: I'm also seeing jobs that supposedly started, but in the log they're waiting to be picked up by runners that don't even exist.

**Reply by jimmybsilva** [reply]
oh man, well I at least now know its not just me and I can go rest and see if they resolve it soon. We hope ...lol

**Comment by cmanfre4** [comment]
Closing the loop on this. It appears to have resolved itself.
